### PR TITLE
Extract jinja_partials 

### PR DESCRIPTION
**Pull request by Colelyman**
This PR removes the dependency on `jinja_partials` and moves part of the `jinja_partials` code into this repo.

### CODE REVIEW

File: CRISPResso2/CRISPRessoCORE.py [hunk 1]
```diff
@@ -150,20 +150,6 @@ def get_n_reads_bam(bam_filename,bam_chr_loc=""):
         raise CRISPRessoShared.InstallationException('Error when running the command:' + cmd + '\nCheck that samtools is installed correctly.')
     return retval
 
-#import time
```
Comment (Snicker7) [inline CRISPResso2/CRISPRessoCORE.py:153]: Is this stuff related to partials? Or being removed for another reason?

**Comment by mbowcut2** [comment]
Passes all tests. Just copies over partials functions. LGTM.

**Review by mbowcut2 — APPROVED** [review]

**Review by Snicker7 — APPROVED** [review]
Looks good me as well.
